### ISSUE TITLE
Update icon colors to meet MAS 1.411

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#115799]
This issue is track all of the issues around icons not meeting the 3:1 contrast ratio. 

From [MAS 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

> For active controls any visual information provided that is necessary for a user to identify that a control is present and how to operate it must have a minimum 3:1 contrast ratio with the adjacent colors. Also, any visual information necessary to indicate state, such as whether a component is selected or focused must also ensure that the information used to identify the control in that state has a minimum 3:1 contrast ratio.

## Default Dark
_None_

## Default Light
- [x] Current line icon `debug-stackframe` fails @ `2:1`
- [x] Extension ratings `extensions-star-full` fails @ `1:4`
- [x] Warning `warning` fails @ `2:1`

FYI @sana-ajani @isidorn

**Comment by isidorn** [comment]
Thanks for creating this. I think this makes sense and we should tackle.

**Comment by misolori** [comment]
For verification, use a [color contrast tool](https://webaim.org/resources/contrastchecker/) to verify that the above icons meet the 3:1 ratio on the default light theme.

**Comment by connor4312** [comment]
btw @misolori devtools' "inspect element" also shows contrast ratio 🙂 

![](https://memes.peet.io/img/21-02-8f2e7aff-0753-4333-9384-a0248a37ab05.png)
